### ISSUE TITLE
Ready isn't successful/unsuccessful - waiting

### ISSUE THREAD

**Issue by mato75** [johntitus/node-horseman#163]
Sometimes it happens that ready is neither successful and unsuccessful, so it is waiting and waiting.

The funny part is, that loadFinishedSetup is triggered, but still, ready is not set.

**Comment by awlayton** [comment]
What is "ready"? I am not sure what you are talking about.

**Comment by mato75** [comment]
.ready in index.js

**Comment by awlayton** [comment]
It is a Promise of a page, not successful/unsuccessful.

**Comment by mato75** [comment]
Well yes, it is not returning a promise so ready is never done. By what I was able to find out is, that about:blank was not opened, so that means that page is in most cases not created ok or it doesnt have enough resources to open the page. I added an interval of 2s, which will try to open about:blank again, and a timeout of 15s, that checks, if page is not created during that time, we return an error.
